src/lib/selection/text-selector.js

```diff
@@ -1,6 +1,5 @@
 import { LanguageModelFactory, TextQuoteSelector } from 'alpheios-data-models'
 
-import HTMLSelector from '@/lib/selection/media/html-selector'
```

Comment:
The same - I was to delete it - otherwise could not build the code.

Reply:
Interesting. The linter? I wonder why it isn't giving me the same error.